### PR TITLE
PROD-264: Guide for import/export template

### PR DESCRIPTION
**Pull request by chrisb-100ms**
https://100ms-docs-git-prod-264-100mslive.vercel.app/docs/get-started/v2/get-started/100ms-dashboard/export-import-template

### CODE REVIEW

**Review by hdz-666 — APPROVED** [review]

File: docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Export Template
+
+![Export Template](/docs/v2/100ms-dashboard/export-template.png)
+
+1. Click on the **Templates** dropdown in the sidebar, and pick a template you want to export 
+1. Once you've selected your template, click on the three dot menu near the top right
+1. From the dropdown, click on **Export Template** and you're done
+
+You should now see exported template.json in your downloads.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx:20]: dont think the name of the file is `template.json`

File: docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+</video>
+
+<Note type='white' title="Note: Select correct workspace and account">
+    Make sure that you are logged in to the correct account and you've selected the correct workspace before importing your template
+</Note>
+
+<br></br>
+
+1. Click on **Create Template**
+1. Click on **Import Template** and choose the template.json you exported in the previous step
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx:35]: same the file is not template.json

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx (hunk 1 above)
Comment (hdz-666) [reply docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx:20]:
```suggestion
You should now see exported <template-name>.json in your downloads.
```

File: docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx (hunk 2 above)
Comment (hdz-666) [reply docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx:35]:
```suggestion
1. Click on **Import Template** and choose the <template-name>.json you exported in the previous step
```

**Review by hdz-666 — APPROVED** [review]

**Review by saikatmitra91 — APPROVED** [review]

**Pull request merged**